mantidproject/mantid
# Adding plot guess to fitting in muon analysis

**Description of work.**
When performing a fit in muon analysis it would be useful to have an option to plot the currently selected function. This PR does that.
It also eliminates the group workspace name option as it is no longer needed.

**To test:**
1. `Interfaces->Muon Analysis 2->Fitting`
2. Check `Plot guess`, nothing should happen
3. Load some data (e.g. `62260`) and check `Plot guess`, nothing should happen
4. Add a function (e.g. `Muon->Abragam`)
5. Check `Plot guess`, a function should appear on the plot with label `guess`
6. Modify the function parameters, the plot should update
7. Fit the data, the guess should disappear

Fixes #26693 
Fixes #26479




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
